src/System.Console/src/PinvokeAnalyzerExceptionList.analyzerdata.uap

```diff
@@ -0,0 +1 @@
+user32.dll!GetKeyState
```

Comment:
This and the process ones are omissions on the Windows side -- driving with email and issues on their side.